diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index f37e3cfc946bc8926b808b1fda9b7a3770810a96..aaca02b2a8e7fe5636856c5e713e7cf4c747037f 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -724,6 +724,7 @@ class BASE_EXPORT [[maybe_unused, nodiscard]] ScopedAllowBaseSyncPrimitives {
   ScopedAllowBaseSyncPrimitives& operator=(
       const ScopedAllowBaseSyncPrimitives&) = delete;
 
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
