diff --git a/base/BUILD.gn b/base/BUILD.gn
index 93ea9dd3196d82544bf20999227b0d4218d65a90..f187f57ba490f24b8439287882a71304822016a2 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4386,6 +4386,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
