diff --git a/build/toolchain/apple/toolchain.gni b/build/toolchain/apple/toolchain.gni
index 93e65289fcc48ec897e887fc7f4d17db2108239f..e287346971e99299e0ccfe5686e6d8e2ad27e5cc 100644
--- a/build/toolchain/apple/toolchain.gni
+++ b/build/toolchain/apple/toolchain.gni
@@ -558,7 +558,7 @@ template("single_apple_toolchain") {
       if (host_os == "mac") {
         command = "rm -rf {{output}} && /bin/cp -Rc {{source}} {{output}}"
       } else {
-        command = "rm -rf {{output}} && /bin/cp -Rl {{source}} {{output}}"
+        command = "rm -rf {{output}} && /bin/cp -Rld {{source}} {{output}}"
       }
       description = "COPY_BUNDLE_DATA {{source}} {{output}}"
       pool = "//build/toolchain/apple:bundle_pool($default_toolchain)"
