diff --git a/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc b/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
index cc5d574692069f4816fc393835ece5c022bdd91c..2a21a4dbb36a2ab88349d3ff36b48a108a3bbf9b 100644
--- a/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
+++ b/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
@@ -226,6 +226,7 @@ PermissionPromptBubbleView::PermissionPromptBubbleView(
        GetVisibleRequests(*delegate.get())) {
     AddRequestLine(request);
   }
+  BRAVE_PERMISSION_PROMPT_BUBBLE_VIEW
 
   absl::optional<std::u16string> extra_text = GetExtraText(*delegate.get());
   if (extra_text.has_value()) {
