diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index 5b51144e9874e6b1a4a90ff418ab9beab105cc05..0edb8d1f3c48b8db843a7f03c3fb176f3860acec 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -938,6 +938,7 @@ void BookmarkBarView::Layout() {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   // Set the visibility of the tab group separator if there are groups and
   // bookmarks.
