diff --git a/chrome/browser/ui/tabs/tab_style.cc b/chrome/browser/ui/tabs/tab_style.cc
index cb37a0eba09a5b9e986c0f939e9a60d882b61389..ff94d8e8afd90c48196fa7043389bb8ff3282ecc 100644
--- a/chrome/browser/ui/tabs/tab_style.cc
+++ b/chrome/browser/ui/tabs/tab_style.cc
@@ -16,7 +16,7 @@ constexpr int kSeparatorThickness = 1;
 
 // Returns the height of the separator between tabs.
 int GetSeparatorHeight() {
-  return ui::TouchUiController::Get()->touch_ui() ? 24 : 20;
+  return ui::TouchUiController::Get()->touch_ui() ? 24 : 24;
 }
 
 class GM2TabStyle : public TabStyle {};
