diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 1401c9328fbd7001dc2689435a9e5dcad42a8cc0..c7ea4d5b229879e2ff7deca1c852c4a913f7c720 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3481,6 +3481,7 @@ const FeatureEntry::FeatureVariation kVcSegmentationModelVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
