diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
index f59eb1a56d7d506b38ba872bc4f2ce0f0a09f6f9..fab0dd808e19360889a4a64c33693cb8a30b0da3 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
@@ -326,6 +326,7 @@ void BookmarkBubbleView::ShowBubble(
       dialog_model_builder.Build(), anchor_view,
       views::BubbleBorder::TOP_RIGHT);
   bookmark_bubble_ = bubble.get();
+  BRAVE_BOOKMARK_BUBBLE_VIEW
   if (highlighted_button)
     bubble->SetHighlightedButton(highlighted_button);
 
