diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 19c13a640d46ce5a20c514c1f08d28061e888cd6..e087560d97d95cae45627b614110d1092a8d507f 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2516,6 +2516,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
