diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index 09b222c24e5120fc51a45387fb9b76d96be01fc8..9af4b66ba7653154678b6b0f5929644530715182 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -690,6 +690,7 @@ void LocationBarView::Layout() {
     }
   };
 
+  BRAVE_LAYOUT_TRAILING_DECORATIONS
   add_trailing_decoration(page_action_icon_container_);
   for (ContentSettingImageView* view : base::Reversed(content_setting_views_)) {
     add_trailing_decoration(view);
