diff --git a/components/password_manager/core/browser/affiliation/hash_affiliation_fetcher.cc b/components/password_manager/core/browser/affiliation/hash_affiliation_fetcher.cc
index 842a3fe44617d818a3424eb75383c5527681a78a..7bd577b4e0acebc2e96fb61ec33aeeae57405760 100644
--- a/components/password_manager/core/browser/affiliation/hash_affiliation_fetcher.cc
+++ b/components/password_manager/core/browser/affiliation/hash_affiliation_fetcher.cc
@@ -54,6 +54,7 @@ HashAffiliationFetcher::~HashAffiliationFetcher() = default;
 void HashAffiliationFetcher::StartRequest(
     const std::vector<FacetURI>& facet_uris,
     RequestInfo request_info) {
+  BRAVE_HASH_AFFILIATION_FETCHER_START_REQUEST
   requested_facet_uris_ = facet_uris;
 
   net::NetworkTrafficAnnotationTag traffic_annotation =
