diff --git a/components/bookmarks/browser/bookmark_model.h b/components/bookmarks/browser/bookmark_model.h
index 87b779af1ecf245e52aa4414f98b89ac6cb04ad9..ec59eaf0506d38528850ef6026e571b0da251d36 100644
--- a/components/bookmarks/browser/bookmark_model.h
+++ b/components/bookmarks/browser/bookmark_model.h
@@ -355,6 +355,7 @@ class BookmarkModel : public BookmarkUndoProvider,
     return weak_factory_.GetWeakPtr();
   }
 
+  BRAVE_BOOKMARK_MODEL_H
  private:
   friend class BookmarkCodecTest;
   friend class BookmarkModelFaviconTest;
