diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index caa97b54735c3c83f9e7c3503d3fc13cf04ce7cf..c9220f1410c6df1e26df26d5266472b3fbaaa9f0 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -53,6 +53,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
