diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 4a3982b46b94e19ed6647c52d70cba8278f3b3b6..d8d30f669dd850866183131c6ad98989f0778069 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -339,6 +339,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps
 
   if ((!is_android || enable_vr) && !is_ios) {
     deps += [ ":vector_icons" ]
