diff --git a/content/browser/webui/web_ui_impl.cc b/content/browser/webui/web_ui_impl.cc
index b1c20e8748fde3b4e0b9cdb015a376e5b551564f..c145b092cfb4a19ec4e3ea46227b552a887c9eb7 100644
--- a/content/browser/webui/web_ui_impl.cc
+++ b/content/browser/webui/web_ui_impl.cc
@@ -70,6 +70,7 @@ WebUIImpl::WebUIImpl(WebContents* web_contents)
       web_contents_(web_contents),
       web_contents_observer_(new WebUIMainFrameObserver(this, web_contents_)) {
   DCHECK(web_contents_);
+  BRAVE_WEB_UI_IMPL
 }
 
 WebUIImpl::WebUIImpl(NavigationRequest* request)
