diff --git a/ui/views/controls/menu/menu_separator.cc b/ui/views/controls/menu/menu_separator.cc
index 8d7368148c649bdf50c39014c5feeb2924408c70..6a0eb4f8858d15850a16be3cee78d28a0c2efc3f 100644
--- a/ui/views/controls/menu/menu_separator.cc
+++ b/ui/views/controls/menu/menu_separator.cc
@@ -46,6 +46,7 @@ void MenuSeparator::OnPaint(gfx::Canvas* canvas) {
   if (type_ == ui::PADDED_SEPARATOR)
     paint_rect.Inset(
         gfx::Insets::TLBR(0, menu_config.padded_separator_left_margin, 0, 0));
+  BRAVE_MENU_SEPARATOR_ON_PAINT
   else if (menu_config.use_outer_border)
     paint_rect.Inset(gfx::Insets::VH(0, 1));
 
