diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index bae2b511af136ee456b56a7bd4c2e64a60a0af88..1e129dc4f1a0e6fe72172a42532f4d29173b7e23 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -117,6 +117,7 @@ generate_grd("build_grd") {
       "$root_gen_dir/ui/webui/resources/cr_components/page_image_service/resources.grdp",
     ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos_ash) {
