diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index c7e5e51bd36764d97d1475896fd85290439922bd..fb0d068feaad24728c05dc6a960b5c07a47d56ff 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2900,6 +2900,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
